src/PdfReader/index.tsx

```diff
@@ -42,6 +47,7 @@ type PdfReaderAction =
     };
 
 const IFRAME_WRAPPER_ID = 'iframe-wrapper';
+export const SCALE_STEP = 0.1;
```

Comment:
nit: should this be in `constants` ?

Reply:
Good question. I think it's specific to this file so fine to leave in here for now personally